.github/workflows/build-test-coverage.yml

```diff
@@ -53,4 +59,24 @@ jobs:
       - name: Unit tests
         run: ./gradlew unit-tests
       - name: Integration tests
-        run: ./gradlew integration-tests
\ No newline at end of file
+        run: ./gradlew integration-tests
+  coverage:
```

Comment:
The coverage job is now bundled in this build-test-coverage shared component, used from our actual workflows (`workflow-*.yml`) except the one for scheduled acceptance tests